src/refmt_args.ml

```diff
@@ -56,6 +56,10 @@ let in_place =
   let doc = "reformat a file in-place" in
   Arg.(value & flag & info ["in-place"] ~doc)
 
+let add_printers =
+  let doc = "add auto-printers to user-defined types" in
+  Arg.(value & flag & info ["a"; "add-printers"] ~doc)
```

Comment:
"-a" option is probably not needed for daily interactive use,  maybe we can remove it.